docs/docs/writing-a-long-lived-client.md

```diff
@@ -2,34 +2,29 @@
 
 ## Prerequisites
 
-This tutorial assumes you have read and understood [Writing your first enclave](writing-hello-world.md).
+Complete the following tutorials.
 
-## Introduction
+* [Running your first enclave](running-hello-world.md).
+* [Writing your first enclave](writing-hello-world.md).
+* [Writing your own enclave host](writing-your-own-enclave-host.md).
 
-The introductory hello-world sample (see [Running your first enclave](running-hello-world.md)
-and [Writing your first enclave](writing-hello-world.md)) implements a scenario where each run is independent, so the
-client does not need to maintain any state between runs: it can generate a new one-time key each time and there is no
-need to track a multi-step protocol.
+## Introduction
 
-Most real-world scenarios are more complex than this. In particular, the interaction between a client and
-enclave may unfold over an extended period of time, meaning the client needs to be able to resume successfully after a
-restart.
+In the [hello-world sample](running-hello-world.md) and [writing your first enclave](writing-hello-world.md) tutorial,
+each run is independent. In such cases, the client doesn't need to maintain any state between runs.
```

Comment:
I would simplify.
`each run is independent, as the client does not maintain any state between runs`

Reply:
I believe the first sentence is the *cause* for the second sentence here. 

In the previous two tutorials, each run is independent. Only as it's independent, the client didn't have to maintain state.

Please correct me if I'm wrong.